### PR TITLE
Returning Insert query

### PR DESCRIPTION
**Pull request by pxp9**

### CODE REVIEW

File: src/schema.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+table! {
```
Comment (ayrat555) [inline src/schema.rs:1]: please remove this file

File: src/asynk/async_queue.rs [hunk 2]
```diff
@@ -284,17 +284,15 @@ where
     pub async fn insert_task_query(
```
Comment (ayrat555) [inline src/asynk/async_queue.rs:284]: can we add a separate test for this query?

**Review by ayrat555 — COMMENTED** [review]
looks great

File: src/asynk/async_queue.rs (hunk 2 above)
Comment (pxp9) [reply src/asynk/async_queue.rs:284]: we have already have it 
``rust
 #[tokio::test]
    async fn insert_task_creates_new_task() {
        let pool = pool().await;
        let mut connection = pool.get().await.unwrap();
        let mut transaction = connection.transaction().await.unwrap();

        let result =
        let task =
            AsyncQueue::<NoTls>::insert_task_query(&mut transaction, &AsyncTask { number: 1 })
                .await
                .unwrap();
        let metadata = task.metadata.as_object().unwrap();
        let number = metadata["number"].as_u64();
        let type_task = metadata["type"].as_str();

        assert_eq!(1, result);
        assert_eq!(Some(1), number);
        assert_eq!(Some("AsyncTask"), type_task);
        transaction.rollback().await.unwrap();
    }
``

**Review by ayrat555 — APPROVED** [review]

**Pull request merged**